server/src/main/java/com/google/coffeehouse/common/StorageHandler.java

```diff
@@ -14,41 +14,110 @@
+        .to(clubId)
+        .set("membershipType")
+        .to(MembershipConstants.MEMBER)
+        .set("timestamp")
+        .to(Value.COMMIT_TIMESTAMP)
+        .build());
+      dbClient.write(mutations);
+  }
+
+  /**
+  * Deletes a membership from the database.
+  * This method checks if a person is already a member of a club. If the person is not a member, it will throw
+  * an exception. Otherwise, it deletes the row containing the user ID and club ID from the Memberships table.
+  *
+  * @param  dbClient    the database client 
+  * @param  userId      the user ID string used to query the membership table
+  * @param  clubId      the club ID string used to query the membership table
+  */
+  public static void deletePersonClubMembershipDml(DatabaseClient dbClient, String userId, String clubId) {
+    if (!StorageHandlerHelper.checkPersonInClubQuery(dbClient, userId, clubId)) {
+      throw new IllegalArgumentException(MembershipConstants.PERSON_NOT_IN_CLUB);
+    }
+    dbClient
+      .readWriteTransaction()
+      .run(
+          new TransactionCallable<Void>() {
+            @Override
+            public Void run(TransactionContext transaction) throws Exception {
+              String sql = "DELETE FROM Memberships WHERE userId = '"
+                    + userId + "' AND clubId = '" + clubId + "'";
+              long rowCount = transaction.executeUpdate(Statement.of(sql));
+              System.out.printf("%d record deleted.\n", rowCount);
+              return null;
+            }
+          });
+  }
+
   /**
-  * Returns a string containing information about a person queried from the database.
-  * This method formats a string with a user's ID, email, nickname, and pronouns.
+  * Creates and returns a {@link Person} with the info about a person queried from the database.
+  * This method builds a {@link Person} with a user's ID, email, nickname, and pronouns.
   * If the user has no pronouns (NULL), or the pronouns are empty, then "No pronouns"
   * is printed out instead.
   *
   * @param  dbClient    the database client 
   * @param  userId      the user ID string used to query and get a person's information
-  * @return personInfo  the formatted string containing the person information
+  * @return person      the Person object built containing the person information
   */
-  public static String getPersonQuery(DatabaseClient dbClient, String userId) {
-    String personInfo = "";
+  public static Person getPersonQuery(DatabaseClient dbClient, String userId) {
     String email = "";
     String nickname = "";
     String pronouns = "";
+    // TODO: implement setting user ID if it already exists @linamontes10
     long resultCount = StorageHandlerHelper.getPersonCountQuery(dbClient, userId);
+    if (resultCount > 1) {
+      throw new IllegalArgumentException(ERROR_MORE_THAN_ONE_PERSON);
+    } else if (resultCount < 1) {
+      throw new IllegalArgumentException(PERSON_DOES_NOT_EXIST);
+    }
     Statement statement = 
```

Comment:
Did you consider using the more declarative read syntax here, instead of SQL? https://cloud.google.com/spanner/docs/reads#perform-strong-read

Reply:
The read syntax is much cleaner, I'll switch the SQL statements to be strong read's instead! Thank you for that suggestion.